config/locales/candidate_interface/gcse_details.yml

```diff
@@ -19,35 +19,40 @@ en:
     page_title: What grade is your %{subject} %{qualification_type}?
     guidance:
       main: |
-        Providers expect you to have a grade C or 4 (or the equivalent) in order
-        to train.
+        You need at least a grade 4 (C).
 
-        If you do not have this, contact your provider to discuss your
-        options. You might be able to retake an exam or do an equivalence test.
+        If you do not have this, contact teacher training providers to discuss your eligibility for their courses. You might be able to retake an exam or do an equivalence test.
       o_level_guidance: |
-        Providers expect you to have a grade C or above in order to train.
+        You need at least a grade C or equivalent.
 
-        If you do not have this, contact your provider to discuss your
-        options. You might be able to retake an exam or do an equivalence test.
-      o_level_triple_gcse_science: If you have a combined or triple O Level in science, enter your total grade.
+        If you do not have this, contact teacher training providers to discuss
```

Comment:
I think `simple_format` is inserting `<br>` for the line breaks here...

<img width="461" alt="image" src="https://user-images.githubusercontent.com/450843/149939939-0f26aa9e-37cd-4c14-acbf-3c7f6ed181dd.png">

Reply:
thanks, fixed